app/controllers/games_controller.rb

```diff
@@ -0,0 +1,18 @@
+class GamesController < ApplicationController
```

Comment:
We will need to add a new action and new view page where users can go and create a game